src/v2/Apps/Article/Components/Sections/ArticleSectionSocialEmbed.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, useEffect, useRef } from "react"
+import { ArticleSectionSocialEmbed_section } from "v2/__generated__/ArticleSectionSocialEmbed_section.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Box } from "@artsy/palette"
+import styled from "styled-components"
+
+interface ArticleSectionSocialEmbedProps {
+  section: ArticleSectionSocialEmbed_section
+}
+
+const ArticleSectionSocialEmbed: FC<ArticleSectionSocialEmbedProps> = ({
+  section,
+}) => {
+  const ref = useRef<HTMLDivElement | null>(null)
+
+  useEffect(() => {
+    if (!ref.current || !section.embed) return
+
+    // React doesn't load the script tags when using `dangerouslySetInnerHTML`
+    // So just inject the HTML manually after mount.
+    ref.current.innerHTML = section.embed
```

Comment:
Previously we were using this in Reaction: https://github.com/humanmade/react-oembed-container —

They specifically say in their readme that this shouldn't work but it ... appears to work fine. Will just revisit if needed.